POC/api/main.py

```diff
@@ -53,6 +54,77 @@ def get_success_button():
     ]
 
 
+@app.get(
+    "/api/forms/create/database",
+    response_model=FastUI,
+    response_model_exclude_none=True,
+    include_in_schema=False,
+)
+async def create_new_database():
+    return [
+        c.Page(
+            components=[
+                c.Heading(text="Dynamic-DB", level=2),
+                c.Paragraph(
+                    text="Welcome! Here you will begin creating a new database."
+                ),
+                c.Markdown(text="---"),
+                c.Div(
+                    components=[
+                        c.Heading(
+                            text="Step 1. Basic Database Information",
+                            level=3,
+                            class_name="text-center",
+                        ),
+                        c.Markdown(text="---"),
+                        c.ModelForm(
+                            loading=[c.Text(text="Submitting")],
+                            model=DbInfoForm,
+                            submit_url="/api/forms/create/database",
+                        ),
+                        c.Markdown(text="---"),
+                        c.ModelForm(
+                            loading=[c.Text(text="Submitting")],
+                            model=AddFieldForm,
+                            method="GET",
+                            submit_url="/api/forms/create/databaseField",
+                        ),
+                    ]
+                ),
+                c.Markdown(text="---"),
+            ]
+        )
+    ]
+
+
+@app.get(
```

Comment:
this returns another form once the user clicks "Add Field" so the user can fill out another field to add to the database.